<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MVP-Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="TestItems">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startTestButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>  开始测试</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="logText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>651</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 14pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLogBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>清空日志信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelComPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(170, 170, 0);</string>
    </property>
    <property name="text">
     <string>COM端口号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="comPortEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>          6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Yes_Btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>620</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="No_Btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>620</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>clearLogBtn</tabstop>
  <tabstop>logText</tabstop>
  <tabstop>TestItems</tabstop>
  <tabstop>comPortEdit</tabstop>
  <tabstop>Yes_Btn</tabstop>
  <tabstop>No_Btn</tabstop>
  <tabstop>startTestButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
